WordPress/Classes/ViewRelated/Reader/WPImageViewController.m

```diff
@@ -5,9 +5,9 @@
 
 @interface WPImageViewController ()<UIScrollViewDelegate>
 
-- (void)handleImageTapped:(UITapGestureRecognizer *)tgr;
-- (void)handleImageDoubleTapped:(UITapGestureRecognizer *)tgr;
-
+@property (nonatomic) BOOL isLoadingImage;
```

Comment:
Should we specify `assign` like we do for `shouldHideStatusBar`?  IIRC the compiler does this automatically but it seems like we were leaning toward making it explicit in the property definitions. Not sure, but I think we should be consistent with how we declare the two BOOLs